admiral/cmd/admiral/cmd/root.go

```diff
@@ -93,6 +93,8 @@ func GetRootCmd(args []string) *cobra.Command {
 		"The parameter will be used to decide whether to update workload sidecar resource or not. By default these updates will be disabled.")
 	rootCmd.PersistentFlags().StringVar(&params.WorkloadSidecarName, "workload_sidecar_name", "default",
 		"Name of the sidecar resource in the workload namespace. By default sidecar resource will be named as \"default\".")
+	rootCmd.PersistentFlags().StringVar(&params.LabelSet.EnvKey, "env_key", "admiral.io/env",
```

Comment:
Should we be more specific on behavior here? The behavior seems to be looking at (in order) annotation with envKey, label with envKey, label with env, pulling from ns. Saying that directly will prevent someone from needing to go through the code to learn it.